test/test_site/expected/siteData.json

```diff
@@ -54,7 +54,9 @@
         "nested-panel-without-src": "Nested panel without src",
         "panel-with-src-from-another-markbind-site": "Panel with src from another Markbind site",
         "modal-with-panel-inside": "Modal with panel inside",
-        "unexpanded-panel": "Unexpanded panel"
+        "unexpanded-panel": "Unexpanded panel",
+        "div-for-plugins": "Div for plugins",
+        "undefined": "Prerender"
```

Comment:
A little odd due to manual adding in the `<h2>` header, but I don't think it's a deal breaker

Reply:
Hmm ok. Probably an existing bug in the way MarkBind collects heading.